runtime-interface/sandbox/src/detail.rs

```diff
@@ -61,13 +61,22 @@ impl Sandboxes {
     }
 }
 
+// Global sandbox backend type selector
+static SANDBOX_BACKEND_TYPE: sandbox_env::AtomicSandboxBackend =
+    sandbox_env::AtomicSandboxBackend::new(sandbox_env::SandboxBackend::Wasmer);
+
 thread_local! {
-    static SANDBOXES: RefCell<Sandboxes> = RefCell::new(Sandboxes::new());
+    static SANDBOXES: RefCell<Sandboxes> = {
+        let sandbox_backend = SANDBOX_BACKEND_TYPE.load(Ordering::SeqCst);
+        RefCell::new(Sandboxes::new(sandbox_backend))
+    }
```

Comment:
It won't work because we need to initialize the sandbox for every OS thread with the sandbox type. In the suggested approach, it will panic on a new thread.